bin/get-github-oauth-token

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+var Github = require('github'),
+    read   = require('read');
+
+read({
+   prompt: "Github Username:"
+}, function(error, response) {
+   getPassword({username: response});
+})
+
+function getPassword(auth) {
+   read({
+      silent: true,
+      prompt: "Github Password:"
+   }, function(error, response) {
+      auth.password = response;
+      getOTP(auth);
+   })
+}
+
+function getOTP(auth) {
+   read({
+      prompt: "2-Factor Auth Code (leave blank for none):"
```

Comment:
Hmm, maybe we should just be checking via the API if they have 2-Factor Auth turned on? https://developer.github.com/v3/auth/